galley/pkg/testing/mock/kube.go

```diff
@@ -31,15 +30,15 @@ type Kube struct {
 	response1 []interface{}
 	response2 []error
 
-	client    kubernetes.Interface
-	clientset clientset.Interface
+	client          kubernetes.Interface
+	APIExtClientSet *fake.Clientset
```

Comment:
Why is it public - there's already an accessor? Can we just make the accessor not return an error? Or should we just get rid of the accessor altogether?

Reply:
The accessor is part of the interface (`client.Interfaces`) and is of the expected type by the prod code. This one is the fake type that implements the interface. It is public here, so that the test code can access the fake part of the functionality.